v1/instances.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 IBM Corp.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package keyprotect
+
+import (
+	"context"
+	"time"
+)
+
+type CreateInstPoliciesInput struct {
+	Instance string `json:"-" location:"header" locationKey:"Bluemix-Instance"`
+
+	Metadata PoliciesMetadata     `json:"metadata"`
+	Policies *[]InstPolicyDetails `json:"resources,omitempty" mapstructure:"resources"`
+}
+
+type InstPolicyDetails struct {
```

Comment:
I think its worth fully naming it InstancePolicy* rather than truncating it.

Reply:
ok